apps/site/pages/en/learn/getting-started/introduction-to-nodejs.md

```diff
@@ -96,3 +96,5 @@ and we close the response, adding the content as an argument to `end()`:
 ```js
 res.end('Hello World\n');
 ```
+
+[Download node.js](https://nodejs.org/en/download/package-manager)
```

Comment:
```suggestion
If you haven't already done so, you can [download](https://nodejs.org/en/download) node.js and start discovering it.
```

Reply:
isn't Node.js always supposed to be capitalized when in writing ?